applications/visualizer/frontend/src/components/viewers/EM/EMStackTilesViewer.tsx

```diff
@@ -84,11 +41,62 @@ const newSegLayer = (dataset: Dataset, slice: number) => {
       url: getSegmentationURL(dataset, slice),
       format: new GeoJSON(),
     }),
-    style: getFeatureStyle,
     zIndex: 1,
   });
 };
 
+function neuronsStyle(feature: FeatureLike, workspace: Workspace) {
+  const neuronName = neuronFeatureName(feature);
+
+  const visibleNeurons = workspace.getVisibleNeuronsInEM();
+  const visibleNeuron = visibleNeurons.find((neuron) => neuron === neuronName);
```

Comment:
In this case, it's better to use `includes` instead of `find` as it's easier to read and to understand. Creating a closure is heavy and its use in `find` also.